src/views/preview/comment/comment.scss

```diff
@@ -172,7 +174,7 @@
                 position: absolute;
                 bottom: 0;
                 background: linear-gradient(
-                    transparent,
+                    $cross-browser-transparent,
```

Comment:
Is the issue that other browsers use transparent white instead of transparent black? Or do they figure out and use the color being transitioned to/from?

I guess my question is for true cross-browser consistency, should define and use `$ui-light-primary-transparent`?

Reply:
As far as I understood they use white transparent instead of dark transparent. 

Hm, in colors.css you mean? I can do that!